util/util_test.go

```diff
@@ -0,0 +1,20 @@
+package util
```

Comment:
please use a separate package for testing, ie: `util_test`. this will allow you to better understand the exposed surface of your package